### PR TITLE
feat: Added new motivational quote

### PR DESCRIPTION
**Pull request by abhayvershwal**
Hello @Shariar-Hasan This repo is perfect for a beginner like me to get started with Hactoberfest. Thank you for creating this repo. Added a quote, please give your suggestions.

### CODE REVIEW

File: Quotes.js [hunk 1]
```diff
@@ -1548,7 +1578,7 @@ window.quotes = [
   {
     quote: "Dream, dream, dream. Dreams transform into thoughts and thoughts result in action",
     category: "Inspiration",
-    author: " Atal Bihari Vajpayee",
+    author: "APJ Abdul Kalam",
```
Comment (abhayvershwal) [inline Quotes.js:1581]: This quote is actually by APJ Abdul Kalam ji. So I have corrected it.

**Comment by abhayvershwal** [comment]
Thank you for your comment @Shariar-Hasan I have updated the PR. Please let me know if anything else is needed.

**Comment by abhayvershwal** [comment]
@Shariar-Hasan Thank you for merging this. Could you please add the Hackcoberfest label to this PR.

**Comment by Shariar-Hasan** [comment]
> @Shariar-Hasan Thank you for merging this. Could you please add the Hackcoberfest label to this PR?

dont worry about it,
the repository have hacktoberfest topiced,
so your merge will autometically counted.
you can check your hacktoberfest profile

**Pull request merged**